### PR TITLE
Add rockspec/rock publishing workflow

### PR DESCRIPTION
**Pull request by ylobankov**

### CODE REVIEW

File: .github/workflows/publishing.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          sed \
+            -e "s/branch = '.\+'/tag = '${{ github.ref_name }}'/g" \
+            -e "s/version = '.\+'/version = '${{ github.ref_name }}-1'/g" \
+            ${{ env.ROCK_NAME }}-scm-1.rockspec > \
+            ${{ env.ROCK_NAME }}-${{ github.ref_name }}-1.rockspec
+      - name: Push rockspec
+        uses: tarantool/rocks.tarantool.org/github-action@master
+        with:
+          auth: ${{ secrets.ROCKS_AUTH }}
+          files: ${{ env.ROCK_NAME }}-${{ github.ref_name }}-1.rockspec
```
Comment (Totktonada) [inline .github/workflows/publishing.yml:53]: I would also publish `.all.rock` to speed up the installation on the user side. See https://github.com/tarantool/expirationd/blob/bcbe8c46ceca8d82dc5bb66b1d0a40deca03ad9d/.github/workflows/publish.yml#L43-L71 for example.

Comment (ylobankov) [reply .github/workflows/publishing.yml:53]: Makes sense, thank you!

Comment (ylobankov) [reply .github/workflows/publishing.yml]: Fixed.

**Review by Totktonada — APPROVED** [review]
Thank you! I see no problems.

**Review by Serpentian — APPROVED** [review]
Thank you for the workflow! No objections

**Review by Gerold103 — APPROVED** [review]
Thanks for the patch!

**Pull request merged**